shell/server/proxy.js

```diff
@@ -1549,6 +1597,10 @@ class Proxy {
         response.setHeader("Set-Cookie", rpcResponse.setCookies.map(makeSetCookieHeader));
       }
 
+      const mainUrl = process.env.ROOT_URL;
+      response.setHeader("Content-Security-Policy", "frame-ancestors " + mainUrl);
```

Comment:
Huh... nevermind. Looks we don't actually set the `frame-src` directive from our code. Maybe Meteor's doing it.